GTMAppAuthSwift/Sources/GTMOAuth2KeychainCompatibility.swift

```diff
@@ -0,0 +1,321 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+import AppAuthCore
+import GTMSessionFetcherCore
+
+// Standard OAuth keys
+let oauth2AccessTokenKey = "access_token"
+let oauth2RefreshTokenKey = "refresh_token"
+let oauth2ScopeKey = "scope"
+let oauth2ErrorKey = "error"
+let oauth2TokenTypeKey = "token_type"
+let oauth2ExpiresInKey = "expires_in"
+let oauth2CodeKey = "code"
+let oauth2AssertionKey = "assertion"
+let oauth2RefreshScopeKey = "refreshScope"
+
+// URI indicating an installed app is signing in. This is described at
+//
+// https://developers.google.com/identity/protocols/OAuth2InstalledApp#formingtheurl
+//
+let oobString = "urn:ietf:wg:oauth:2.0:oob"
+
+/// Class to support serialization and deserialization of @c GTMAppAuthFetcherAuthorization
```

Comment:
Should we be using backticks for symbol references in doc comments?  Looks like you've got a mix of `@c` and `` ` ` `` here.

Reply:
Ah yes. Good catch.